src/social-network-provider/twitter.com/encoding.ts

```diff
@@ -51,7 +50,7 @@ export const twitterEncoding = {
      * @link https://github.com/DimensionDev/Maskbook/issues/198
      */
     payloadEncoder: (text: string) =>
-        `https://${randomElement(topSites)}/${batchReplace(text, [
+        `https://maskbook.com/?PostData_v1=${batchReplace(text, [
```

Comment:
Why?

Reply:
discussed already and will keep this design. users feel random link to huge co. are scammy